apps/gitness/src/components/breadcrumbs/breadcrumbs-mfe.tsx

```diff
@@ -0,0 +1,57 @@
+import { Link, useMatches } from 'react-router-dom'
+
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator,
+  Topbar
+} from '@harnessio/ui/components'
+
+import { CustomHandle } from '../../routes'
+
+function BreadcrumbsMFE({ selectedProject }: { selectedProject: string }) {
```

Comment:
@vardanbansal-harness can you review this change? do we need a complete separate copy of the breadcrumb component? or could we manage this with just the new routes + handles?

Reply:
Agreed, I don't see the need for it. All it does, in addition to what we already have, is render `selectedProject` as a fixed breadcrumb, which I think can be managed in the existing component we have.

`<span className="ml-1">{selectedProject}</span>`